### PR TITLE
Include useAuth hook in Login Page to handle states + injecting header details

### PR DESCRIPTION
**Pull request by AlexisRoe**
including the useAuth hook, to handle 

- [x] error states including the form, if password is wrong or user is not found
- [x] loading state of the notification area
- [x] preparitions for including jwt/ excluding jwt cookie

### CODE REVIEW

File: client/src/Pages/Login.js [hunk 1]
```diff
@@ -1,41 +1,67 @@
...
-      <LoginButton type="submit">Login</LoginButton>
-    </LoginForm>
+      <Main>
+        <LoginForm onSubmit={handleSubmit}>
+          <LoginInput
+            type="text"
+            value={credentials.id}
+            title="Mitarbeiter-ID"
+            error={formState}
+            onChange={(event) => handleOnChangeID(event.target.value)}
```
Comment (ncbecker) [inline client/src/Pages/Login.js:50]: I would name the function handleChangeID (along the lines of onSubmit={handleSubmit})

File: client/src/Pages/Login.js [hunk 2]
```diff
@@ -1,41 +1,67 @@
...
+          />
+          <LoginInput
+            type="password"
+            value={credentials.password}
+            title="Password"
+            error={formState}
+            onChange={(event) => handleOnChangePassword(event.target.value)}
```
Comment (ncbecker) [inline client/src/Pages/Login.js:57]: Same as above

**Review by ncbecker — APPROVED** [review]
lgtm 🚀

**Comment by AlexisRoe** [comment]
thanks @ncbecker for reviewing

File: client/src/Pages/Login.js (hunk 1 above)
Comment (AlexisRoe) [reply client/src/Pages/Login.js]: hmmm, okay true

**Review by AlexisRoe — COMMENTED** [review]
renaming accepted 😉

**Pull request merged**